src/custom/pages/App/AppMod.tsx

```diff
@@ -35,14 +35,32 @@ import ReferralLinkUpdater from 'state/affiliate/updater'
 import ApeModeQueryParamReader from 'hooks/useApeModeQueryParamReader'
 
 import Footer from 'components/Footer'
-import { BodyWrapper } from '.' // mod
+import { BodyWrapper } from '.'
+import * as CSS from 'csstype' // mod
 
-const AppWrapper = styled.div`
+interface AppWrapProps {
+  bgBlur?: boolean
+}
+
+const AppWrapper = styled.div<Partial<CSS.Properties & AppWrapProps>>`
   display: flex;
   flex-flow: column;
   align-items: flex-start;
   min-height: 100vh;
   overflow-x: hidden;
+  &:after {
+    content: '';
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    filter: blur(5px);
```

Comment:
Maybe:
filter: blur(10px);
cc @biocom?

Reply:
20px is better.